tests/autoyast/installation.pm

```diff
@@ -16,6 +16,7 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils qw(clear_console);
 
```

Comment:
why not just use utils? We try to make our test cases really simple perl, so what is the problem you forsee by just importing all utils?